base/reflection.jl

```diff
@@ -488,7 +488,7 @@ function uncompressed_ast(m::Method, s::CodeInfo)
 end
 
 # Printing code representations in IR and assembly
-function _dump_function(f::ANY, t::ANY, native::Bool, wrapper::Bool, strip_ir_metadata::Bool, dump_module::Bool)
+function _dump_function(f::ANY, t::ANY, native::Bool, wrapper::Bool, strip_ir_metadata::Bool, dump_module::Bool, asmvariant=0)
```

Comment:
Why not `asmvariant::String="att"`? Being consistent about the default value and further restrict the type to avoid the ambiguity problem.